packages/vuetify/src/mixins/data-iterable.js

```diff
@@ -474,7 +474,9 @@ export default {
       return [this.$createElement('div', {
         'class': this.actionsClasses
       }, [
-        this.$slots['actions-prepend'] ? this.$createElement('div', {}, this.$slots['actions-prepend']) : null,
+        this.$slots['actions-prepend'] ? this.$createElement('div', {
+          'class': 'flex'
```

Comment:
We can't use .flex, it won't work unless the user has loaded it themselves. This should be its own table-specific class and css.